test/check-exercises.py

```diff
@@ -22,7 +25,14 @@ def check_assignment(name, test_file):
     example_name = modname_heuristic(test_file)
     try:
         test_file_out = os.path.join(workdir, os.path.basename(test_file))
-        shutil.copyfile(test_file, test_file_out)
+        if name in ALLOW_SKIP:
+            shutil.copyfile(test_file, test_file_out)
+        else:
+            with open(test_file, 'r') as src_file:
+                lines = [line for line in src_file.readlines()
+                         if not line.strip().startswith('@unittest.skip(')]
```

Comment:
I would suggest removing the opening bracket from the `startswith` argument (ie. make it `'@unittest.skip'`) - `@unittest.skip` can be used without arguments, so this risks not removing those skips and therefore causing problems in future.